kythe/go/extractors/gcp/examples/BUILD

```diff
@@ -6,3 +6,12 @@ go_binary(
     name = "helloworld",
     srcs = ["helloworld.go"],
 )
+
+go_binary(
+    name = "rest_test",
+    srcs = ["rest_test.go"],
```

Comment:
Maybe use a different name than *_test.go since that is usually used for test files. Maybe rest_verification? rest_check?

Reply:
yup thanks, I forgot _test.go is reserved